README.md

```diff
@@ -29,6 +29,22 @@ A list of every a11y concern Checka11y.css will check for and highlight:
 - Missing `lang` on `<html>`
 - Checks `<a target="_blank">` and warns about links that open in a new window
 - Checks `<dt>` and `<dd>` are the **only** children of `<dl>`
+- Checks if following tags are nested inside of `<a>` and warns about them
```

Comment:
Could we change this to "Checks for the following invalid HTML elements nested inside of an `<a>`:"